papers/ice_nucleation_2024/calibration.jl

```diff
@@ -9,6 +9,8 @@ import ClimaParams as CP
 import CloudMicrophysics as CM
 import CloudMicrophysics.Parameters as CMP
 import Thermodynamics as TD
+using StatsBase
```

Comment:
Could we do it as 

`import StatsBase` as SB`

and then

`SB.function(...)` 

when we use it in the code? This way it's easier to read and tell where different functions are defined